.github/workflows/buildAndTestAieTools.yml

```diff
@@ -155,6 +155,8 @@ jobs:
               export LIT_FILTER="${{ inputs.LIT_FILTER }}"
             fi
 
+            export LIT_OPTS="-sv --timeout 300 -j1 --time-tests --order=random"
```

Comment:
Needed this because the runner ran out of memory without it. At least I think that's what happened...it kept getting killed partway through.